lib/record-serialization-fallback.js

```diff
@@ -261,8 +258,10 @@ JsrsParser.prototype.skipComments = function() {
 // Skip the body of a single-line comment (i.e. the part after //)
 //
 JsrsParser.prototype.skipLineCommentBody = function() {
-  while (!this.isNewline(this.lookahead()) &&
-          this.lookahead() !== undefined) {
+  while (
```

Comment:
I think we must decompose all conditions into variables and reduce conditions to single line where it is possible. [See new version](https://github.com/metarhia/jstp/pull/244/commits/0e267a7a054367d6a6e9845962acdcb6fa078830#diff-28f1518a08659d32c7993acb6908fb8eR260)